kgl/src/commonMain/kotlin/com/danielgergely/kgl/Util.kt

```diff
@@ -0,0 +1,13 @@
+package com.danielgergely.kgl
+
+public fun FloatArray.vSize(vecSize: Int): Int {
```

Comment:
I couldn't make these `internal` (I guess because kotlin modules don't cross platform boundaries) so unfortunately this would become part of the kgl public API. I can make them private on each platform if you prefer.